pool_automation/roles/aws_manage/tasks/post_up.yml

```diff
@@ -5,6 +5,16 @@
     dest: "{{ ssh_dir }}/ssh_config"
 
 - name: Create inventory file for AWS hosts
+  vars:
+    ssh_private_key: "{% raw %}{{ inventory_dir }}{% endraw %}/{{ ssh_dir_name }}/key"
+    known_hosts_file: "{% raw %}{{ inventory_dir }}{% endraw %}/{{ ssh_dir_name }}/known_hosts"
```

Comment:
Is `{% raw %}` really needed here?

Reply:
yes, to keep jinja2 expression after rendering. it's an original behavior, no changes